source

```diff
@@ -102671,10 +102678,12 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
    <dd>Switch to the <span>data state</span>. Emit the current tag token.</dd>
 
    <dt>EOF</dt>
-   <dd><span>Parse error</span>. Emit an end-of-file token.</dd>
+   <dd><span data-x="parse-error-eof-in-attribute-value">eof-in-attribute-value</span><span>parse
+   error</span>. Emit an end-of-file token.</dd>
 
    <dt>Anything else</dt>
-   <dd><span>Parse error</span>. <span>Reconsume</span> in the <span>before attribute name
+   <dd>This is a <dfn data-x="parse-error-missing-whitespace-after-attribute-value">missing-whitespace-after-attribute-value</dfn>
```

Comment:
My bad, after some consideration I think it's not a quite good name, it implies where always should be whitespace after attribute value, however it's not true if following character is `>` or `/`. Therefore,
I guess better will be `missing-whitespace-between-attributes`. What do you think?